packages/edit-site/src/components/header-edit-mode/document-actions/style.scss

```diff
@@ -10,7 +9,7 @@
 	// See https://dev.w3.org/csswg/css-flexbox/#min-size-auto
 	min-width: 0;
 	background: $gray-100;
-	border-radius: 4px;
+	border-radius: $grid-unit-05;
```

Comment:
Interesting, I could swear that box was 2px.

Since we're also using this border radius for modal dialogs (`.components-modal__frame`), should we make it a canonized variable like we have one for $radius-block-ui, and apply across both? Or is there a better system for handling the 3 radiuses we use at the moment (2, 4, 8)?

Reply:
Yes a little down the road I think we should consider this. Probably part of the upcoming component theming work.